data/ships.txt

```diff
@@ -226,7 +224,8 @@ ship "Bactrian"
 	explode "large explosion" 45
 	explode "huge explosion" 30
 	"final explode" "final explosion large"
-	description "The Lionheart Bactrian is the last of the great city-ships, a design hearkening back to the early days of space colonization when a long-distance vessel needed to be a self-contained world, able to survive at times for weeks without encountering an inhabited planet. It is not only a freighter, but a carrier, and a capable warship either at short range or at a distance. Naturally, this versatility also makes it extremely expensive, and the Bactrian is not normally for sale to ordinary citizens who have not been vetted by the local government of the Deep."
+	description "The Lionheart Bactrian is the last of the great city-ships, a design hearkening back to the early days of space colonization when a long-distance vessel needed to be a self-contained world, able to survive at times for weeks without encountering an inhabited planet."
+	description "During the caravan era of the 27th century, the Bactrian was among the most powerful warships in human space, as well as being one of the best freighters and carriers. However, more modern times have seen the Bactrian become among the weakest of the warships, and it was not too many years ago that the Bactrian was recategorized as a transport vessel, and as such the Deep no longer requires for those who wish to purchase this massive ship to be vetted."
```

Comment:
As for the meaning of vet, you _can_ check a thing (I stand corrected), but you wouldn't provide "expert appraisal or correction" to a sale (why would you "correct" a sale?), you would "evaluate" someone "for possible approval or acceptance" (the meaning of "vet" when applied to a person, according to one of your sources). Furthermore, I would stick with the usage that the current version uses. It's not incorrect, that's for sure.

Regardless, I think your edit makes the sentence a lot more elegant.
(By the way, there should be a *the in between "controlled by" and "local government".)

Here's my revised version of originally @Pointedstick's, edited by @Tadrix and myself, change to @Amazinite's pull.
 
>During the caravan era of the 27th century, heavily-armed Bactrians were among the most powerful warships in human space, and sales were tightly controlled by the local government of the Deep. No longer a match for today's modern warships, the Bactrian has been reclassified as a Transport and is now sold freely to civilians.